test/quantization/test_quantize_script.py

```diff
@@ -2461,14 +2461,19 @@ def __init__(self):
             def forward(self, x):
                 x = x + 5
                 return self.fc1(x)
-
-        m = torch.jit.script(M())
-        qconfig_dict = {'' : default_dynamic_qconfig}
-        model = quantize_dynamic_script(m, qconfig_dict)
-        # add op is not dynamically quantized.
-        FileCheck().check("aten::add") \
-                   .check("quantized::linear_dynamic") \
-                   .run(model.graph)
+        eager_model = M().eval()
+        for tracing_mode in [True, False]:
+            if tracing_mode:
+                x = torch.randn(5, 5)
+                model = torch.jit.trace(eager_model, x)
+            else:
+                model = torch.jit.script(eager_model)
```

Comment:
we have a utility function for this: `get_script_module`. please take a look at other tests in the file.